azure-pipelines.yml

```diff
@@ -37,13 +37,20 @@ variables:
   - name: analysisProject
     value: OMF_API_CPP
 
+parameters:
+  - name: containerDemandsADH
+    default: SKU -equals VSEnterprise
+  - name: containerDemandsEDS
+    default: SKU -equals VSEnterprise
+  - name: containerDemandsOnPrem
+    default: SKU -equals VSEnterprise
+
+
 jobs:
   - job: Tests_ADH
     pool:
       name: 00-OSIManaged-Build
-      demands:
-      - Agent.OS -equals Windows_NT
```

Comment:
I was unsure why the OS demand was present, since the sample requires the C++ build tools in VS Enterprise in order to run probably, this necessitates that the demanded SKU is Windows based, making the first demand redundant.